app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_post_comments_counter(post)
+    post.update(comments_counter: Comment.where(post_id: post.id).length)
+  end
```

Comment:
- Please remove the parameter `post` from the update_post_likes_counter method. You already defined the `post` in line 3.
- A faster way to update the comment_counter would be:
```ruby
def update_post_comments_counter
  post.increment!(:comments_counter)
end
```